analysers/analyser_merge_cemetery_FR.py

```diff
@@ -33,7 +33,7 @@ def __init__(self, config, logger = None):
         self.init(
             "https://ign.fr",
             "IGN-Cimetière",
-            GPKG(Source(attribution = "IGN", millesime = "09/2020", gzip = True,
+            GPKG(SourceIGN(attribution = "IGN", gzip = True,
```

Comment:
It is required for insert the millesime into the attribution string
https://github.com/osm-fr/osmose-backend/blob/master/analysers/Analyser_Merge.py#L424

```
attribution = "IGN, {0}",
```

Reply:
I tried your proposal, the effect is that it changes the format of the resulting XML file: 
```
-<tag action="create" k="source" v="IGN - 2021-06" />
+<tag action="create" k="source" v="IGN, 2021-06" />
```

So, what is the preferred version?

Reply:
Hum, may be I'm wrong, and no change is required.

Reply:
Anything else needed from my side?